skimage/color/colorconv.py

```diff
@@ -679,6 +679,85 @@ def rgb2xyz(rgb):
     return _convert(xyz_from_rgb, arr)
 
 
+
+def rgb2c1c2c3(rgb):
+    """RGB to C1C2C3 color space conversion.
```

Comment:
I would welcome a short decribtion of what the C1C2C3 is... giving reference is nice but keep it easy to use :)